ui-admin/src/study/participants/enrolleeView/EnrolleeView.tsx

```diff
@@ -70,89 +85,99 @@ export default function EnrolleeView({ enrollee, studyEnvContext, onUpdate }:
     consentMap[configConsent.consentForm.stableId] = { consent: configConsent, responses: matchedResponses }
   })
 
-  return <div className="ParticipantView container mt-4">
+  const navListItemStyle = {
+    backgroundColor: '#ededed',
+    marginBottom: '0.25em',
+    padding: '0.5em'
+  }
+
+
+  return <div className="ParticipantView mt-3 ps-4">
+    <NavBreadcrumb value={enrollee?.shortcode || ''}>
+      <Link to={`${currentEnvPath}/participants/${enrollee.shortcode}`}>
+        {enrollee?.shortcode}</Link>
+    </NavBreadcrumb>
     <div className="row">
       <div className="col-12">
         <h4>
           {enrollee.profile.givenName} {enrollee.profile.familyName} &nbsp;
           <span className="detail" title="Participant shortcode"> ({enrollee.shortcode})</span>
         </h4>
-        <div>
-
-        </div>
       </div>
     </div>
     <div className="row mt-2">
       <div className="col-12">
         <div className="d-flex">
-          <div style={{ minWidth: '280', maxWidth: '280px' }}>
-            <ul className="list-group">
-              <li className="list-group-item">
+          <div style={{ minWidth: '290', maxWidth: '290px' }}>
```

Comment:
```suggestion
          <div style={{ minWidth: '290px', maxWidth: '290px' }}>
```

`minWidth` is being ignored without a unit, which causes the width of the sidebar to suddenly change as sections are collapsed